### PR TITLE
[new] send grafana emails

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Validated and tested that you can send emails for a [grafana service on MST](https://app.intercom.com/a/apps/uvcadzno/articles/articles/2752501/show).

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: mst/aiven-client/grafana-email.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Send Grafana emails
+excerpt: Configure the Simple Mail Transfer Protocol (SMTP) server in MST for Grafana.
+product: mst
+---
+
+# Send Grafana emails
+
+Use the Aiven client to configure the Simple Mail Transfer Protocol (SMTP)
+server settings and send the following emails from Managed Service for
```
Comment (charislam) [inline mst/aiven-client/grafana-email.md:10]:
```suggestion
server settings and send emails from Managed Service for
```

File: mst/aiven-client/grafana-email.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+TimescaleDB for Grafana such as invite emails, reset password emails, and alert
```
Comment (charislam) [inline mst/aiven-client/grafana-email.md:11]:
```suggestion
TimescaleDB for Grafana, including invite emails, reset password emails, and alert
```

File: mst/aiven-client/grafana-email.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+messages.
+
+## Before you begin
+
+*   Installed [Aiven Client][aiven-client-mst].
+*   Signed in to your [Managed Service for TimescaleDB dashboard][mst-login].
```
Comment (charislam) [inline mst/aiven-client/grafana-email.md:17]:
```suggestion
*   Install [Aiven Client][aiven-client-mst].
*   Sign in to your [Managed Service for TimescaleDB dashboard][mst-login].
```

File: mst/aiven-client/grafana-email.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+*   Create a service for [Grafana in MST][grafana-install]
+*   Make a note of these values in the SMTP server: `IP or hostname`, SMTP
```
Comment (charislam) [inline mst/aiven-client/grafana-email.md:19]: Is there a reason SMTP server port isn't formatted as inline code?

File: mst/aiven-client/grafana-email.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    ```bash
+       avn service update --project <PROJECT> <SERVICE_NAME>\
+       -c smtp_server.host=smtp.example.com \
+       -c smtp_server.port=465 \
+       -c smtp_server.username=emailsenderuser \
+       -c smtp_server.password=emailsenderpass \
+       -c smtp_server.from_address="grafana@yourcompany.com"
+    ```
+
+1.  (*OPTIONAL*)Review all available custom options, and configure:
```
Comment (charislam) [inline mst/aiven-client/grafana-email.md:62]:
```suggestion
1.  (*OPTIONAL*) Review all available custom options, and configure:
```

**Review by charislam — APPROVED** [review]

File: mst/aiven-client/grafana-email.md (hunk 5 above)
Comment (Loquacity) [reply mst/aiven-client/grafana-email.md:62]:
```suggestion
1.  *OPTIONAL* Review all available custom options, and configure:
```

**Review by jfjoly — APPROVED** [review]
LGTM

**Pull request merged**